### PR TITLE
Doc 6634: Fedramp for logs and metrics going ga

### PR DESCRIPTION
**Pull request by zuluecho9**
For details, see jira. Synopsis: 

Hidden beta doc about fedramp-compliant logs and dimensional metric data (Log API and Metric API endpoints) is going live and being incorporated into main fedramp doc. While I was at it, also improved structure of the main doc, separating it into a) agents section, and b) data ingest APIs section.

### CODE REVIEW

File: src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx [hunk 1]
```diff
@@ -426,12 +487,26 @@ To ensure FedRAMP compliance, all traffic reporting to `insights-collector.newre
...
+* Fluentbit: Use [our Fluentbit endpoint configuration](https://github.com/newrelic/newrelic-fluent-bit-output#configuration-parameters).
+* Fluentd: Use [our Fluentd endpoint instructions](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/fluentd-plugin-log-forwarding#instrument-plugin).
+* Infrastructure agent: See [FedRAMP for infrastructure](/#infrastructure-endpoints).
+* Kubernetes: Our Kubernetes integration logs are based on fluentbit’s output plugin. Use [these endpoint instructions](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging#installation).
+* Logstash: Use [our Logstash endpoint configuration](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/logstash-plugin-log-forwarding#instrument-plugin).
+* Syslog: For configuring syslog clients, see [TCP endpoint configuration](/docs/logs/log-management/log-api/use-tcp-endpoint-forward-logs-new-relic/).
+* S3: Not yet available.
+* Vector: Not yet available.
+
+To use the Log API directly, you'd edit the [the Log API endpoint configuration](/docs/logs/log-management/log-api/introduction-log-api#endpoint).
```
Comment (jsgithub1) [inline src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx:508]: either remove the 'the' after you'd edit or delete it from the link.  so you don't get "the the"

File: src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx [hunk 2]
```diff
@@ -12,27 +12,35 @@ redirects:
...
+
+New Relic customers must meet all of the following requirements for New Relic’s FedRAMP environment:
+
+  1. **New Relic-approved customers**: New Relic’s FedRAMP-Moderate authorized environment is only available to New Relic-approved customers. For more information, contact your New Relic account representative.
+
+  2. **Order form**: Customer’s order form with New Relic must include customer’s eligibility for FedRAMP.
+
+  3. **Subscription level**: Customer must have a current and valid subscription to New Relic Full-Stack Observability Enterprise or New Relic-approved subscription.
+
+  4. **Authorized New Relic endpoints**: Customer must send its data only to [New Relic’s FedRAMP-designated endpoints](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints).
```
Comment (paperclypse) [inline src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx:25]: "must only"

Comment (paperclypse) [reply src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx:25]: Also, this link is circular and links back to this page. Not sure why we need this link?

Comment (zuluecho9) [reply src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx:25]: touche

File: src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx [hunk 3]
```diff
@@ -193,7 +181,7 @@ In the following table:
 
     <tr>
       <td>
-        [Log management](#info-circle)
+        Logging (with exception of [log patterns](https://docs.newrelic.com/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta/))
```
Comment (paperclypse) [inline src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx:184]: You may want to make this a relative link.

**Review by paperclypse — APPROVED** [review]
Looks good. Had a couple of minor comments, but I don't see them as blockers to merging this in.

**Comment by paperclypse** [comment]
I checked all of the links, because we've been having problems with those. They're all good.

**Pull request merged**